### PR TITLE
Create readme

### PR DESCRIPTION
**Pull request by esdrasedu**
issue #11

### CODE REVIEW

**Comment by leorog** [comment]
+1

**Comment by lucca65** [comment]
+ 1

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+Tasks
+-----
+### Configuration ###
+Kiik gem use config file `config/kiik.yml` for take api_key
+For create this file in your project use task:
+
+`rake kiik:config`
+
+Dont forgete change `api_key` to your key
```
Comment (jonatandahora) [inline README.md]: Grammar Nazi  here...
Don't forget to change `api_key` to your key

File: lib/tasks/kiik.rake [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  desc 'Create config file for kiik'
+  task :config do
+    relative_config = 'config/kiik.yml'
+    src = File.expand_path("../../../#{relative_config}", __FILE__)
+    dst = File.expand_path(relative_config, ENV["BUNDLE_GEMFILE"])
+    unless File.exist? dst
+      FileUtils.cp src, dst
+      puts "Config file created"
+    else
+      puts "Config file ready exists"
```
Comment (jonatandahora) [inline lib/tasks/kiik.rake]: "Config file already exists"

**Comment by jonatandahora** [comment]
+1

**Pull request merged**